app/src/main/java/com/google/samples/apps/sunflower/adapters/SunflowerPagerAdapter.kt

```diff
@@ -17,31 +17,24 @@
 package com.google.samples.apps.sunflower.adapters
 
 import androidx.fragment.app.Fragment
-import androidx.fragment.app.FragmentManager
-import androidx.fragment.app.FragmentStatePagerAdapter
+import androidx.viewpager2.adapter.FragmentStateAdapter
 import com.google.samples.apps.sunflower.GardenFragment
 import com.google.samples.apps.sunflower.PlantListFragment
-import kotlin.IndexOutOfBoundsException
 
 const val MY_GARDEN_PAGE_INDEX = 0
 const val PLANT_LIST_PAGE_INDEX = 1
 
 class SunflowerPagerAdapter(
-    private val tabTitles: HashMap<Int, String>,
     private val tabFragments: HashMap<Int, Fragment>,
-    fragmentManager: FragmentManager
-) : FragmentStatePagerAdapter(fragmentManager, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
+    fragment: Fragment
+) : FragmentStateAdapter(fragment) {
 
     /**
      * There are only two fragments in this ViewPager: [GardenFragment] and [PlantListFragment]
      */
-    override fun getCount() = 2
+    override fun getItemCount() = 2
```

Comment:
Maybe return the number of keys in the tabFragments map?

Reply:
Done.  Also refactored `tabFragments` into `SunflowerPagerAdapter`